docs/content/latest/deploy/multi-dc/_index.html

```diff
@@ -21,27 +21,42 @@
 <p>
 
 <div class="row">
-    <div class="col-12 col-md-6 col-lg-12 col-xl-6">
+
+  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
     <a class="section-link icon-offset" href="3dc-deployment/">
       <div class="head">
-        <img class="icon" src="/images/section_icons/explore/planet_scale.png" aria-hidden="true" />
+        <img class="icon" src="/images/section_icons/explore/planet_scale.png"  aria-hidden="true" />
         <div class="title">Three+ data centers (3DC)</div>
       </div>
       <div class="body">
-        Deploy a single cluster across 3 or more DCs with global consistency (based on synchronous replication using Raft).
+        Deploy a single cluster across 3 or more DCs with global consistency  (based on synchronous replication using Raft).
       </div>
     </a>
   </div>
 
   <div class="col-12 col-md-6 col-lg-12 col-xl-6">
     <a class="section-link icon-offset" href="2dc-deployment/">
       <div class="head">
-        <img class="icon" src="/images/section_icons/explore/planet_scale.png" aria-hidden="true" />
+        <img class="icon" src="/images/section_icons/explore/planet_scale.png"  aria-hidden="true" />
         <div class="title">Two data centers (2DC)</div>
       </div>
       <div class="body">
-        Deploy 2 clusters in 2 DCs is master-follower or multi-master configuration (based on asynchronous replication using CDC).
+        Deploy 2 clusters in 2 DCs is master-follower or multi-master  configuration (based on asynchronous replication using CDC).
       </div>
     </a>
   </div>
+
+  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
+    <a class="section-link icon-offset" href="read-replica-clusters/">
+      <div class="head">
+        <img class="icon" src="/images/section_icons/explore/planet_scale.png" aria-hidden="true" />
+        <div class="title">Read replica clusters</div>
+      </div>
+      <div class="body">
+        Deploy a read replica cluster to asynchronously replicate data from a primary cluster.
```

Comment:
Should be "Deploy a read replica cluster to serve low-latency reads from a remote DC"